apps/condo/domains/ticket/components/BaseTicketForm/TicketAssignments.tsx

```diff
@@ -69,7 +69,13 @@ const TicketAssignments = ({
     const { assignee: initialAssignee, executor: initialExecutor } = form.getFieldsValue(['assignee', 'executor'])
 
     const { objs: employees, allDataLoaded: allEmployeesLoaded } = OrganizationEmployee.useAllObjects({
-        where: { organization: { id: organizationId }, isBlocked: false, isRejected: false },
+        where: {
+            organization: { id: organizationId },
+            user: { deletedAt: null },
+            deletedAt: null,
```

Comment:
Isn't `deletedAt` prop passed to the hook by default?

Reply:
if we manually set deletedAt (in user), then it's will stop being passed automatically